### PR TITLE
chore(appeals): separate codebases

### PR DESCRIPTION
**Pull request by fabfab-kc**
## Describe your changes

Moved appeals web and api to new workspaces. 

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [x] Other (please explain in the description section above)

## Checklist before requesting a review

- [ ] I have performed a self-review of my own code
- [ ] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [ ] I have provided details on how I have tested my code
- [ ] I have referenced the ticket number above
- [ ] I have provided a description of the ticket
- [ ] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: apps/api/index.d.ts [hunk 1]
```diff
@@ -3,4 +3,4 @@ import * as Schema from './src/database/schema';
 
 export { Appeals, Schema };
 
-export type CaseType = 'application' | 'document';
+export type CaseType = 'appeals' | 'document';
```
Comment (beejjacobs) [inline apps/api/index.d.ts:6]: Did you meant to change this @fabfab-kc ?

File: appeals/api/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,23 @@
+FROM node:16-alpine3.16
+WORKDIR /src/app
+COPY package*.json ./
+
+COPY packages ./packages
+
+COPY apps/api apps/api
```
Comment (beejjacobs) [inline appeals/api/Dockerfile:7]: these will need updating - but we can check all of that as part of CI/CD

File: appeals/web/Dockerfile
Comment (beejjacobs) [inline appeals/web/Dockerfile:1]: Needs updating too

File: appeals/api/package.json [hunk 3]
```diff
@@ -0,0 +1,86 @@
+{
+  "name": "@pins/appeals.api",
+  "version": "0.0.0",
+  "private": true,
+  "description": "PINS Back Office API App",
```
Comment (beejjacobs) [inline appeals/api/package.json:5]: `PINS Back Office Appeals API` ?

File: apps/api/index.d.ts (hunk 1 above)
Comment (fabfab-kc) [reply apps/api/index.d.ts:6]: nope, well spotted.

Comment (fabfab-kc) [reply apps/api/index.d.ts]: merging this with main will be fun...

**Comment by beejjacobs** [comment]
Superseded https://github.com/Planning-Inspectorate/back-office/pull/1040

**Pull request closed without merging**